scripts/validate-orbs.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -eo pipefail
+
+for ORB in src/*; do
+  echo "Validating $ORB ..."
+  circleci orb validate $ORB/orb.yml
```

Comment:
You can add `circleci orb process $ORB/orb.yml`, but consider this optional.  I pulled this branch and tried to break an orb. Just tried a couple quick modifications and nothing that passed `validate` failed on `process`.

Reply:
Why do you want to run `process `?

Reply:
At least for `circleci config process`, it did some additional verification over `circleci config validate`: it would fail of referenced orbs didn't exist.  I assume the same is true for `circleci orb process`, but I haven't confirmed.